docs/en/Processing-Request.rst

```diff
@@ -0,0 +1,347 @@
+******************
+Processing Request
+******************
+
+Standard Request
+================
+
+Alexa communicates with the skill service via a request-response mechanism using HTTP over SSL/TLS. When a user interacts with an Alexa skill, your service receives a POST request containing a JSON body. The request body contains the parameters necessary for the service to perform its logic and generate a JSON-formatted response. Since Node.js can handle JSON natively, ASK SDK v2 for Node.js doesn't need to do JSON serialization and deserialization. The documentation on JSON structure of the request body can be found `here <https://developer.amazon.com/docs/custom-skills/request-and-response-json-reference.html#request-format>`_.
+
+Handler Input
+=============
+
+Request handlers, request and response interceptors, and error handlers are all passed a ``HandlerInput`` object when invoked. This object exposes various entities useful in request processing, including:
+
+-  **RequestEnvelope**: Contains the entire `request body <https://developer.amazon.com/docs/custom-skills/request-and-response-json-reference.html#request-body-syntax>`_ sent to skill
```

Comment:
Either add full stops or remove full stops for each item, at the end of the sentence.